### PR TITLE
Issue #6977: Renamed 'reset' command in Appium to 'resetApp'

### PR DESCRIPTION
**Pull request by srikanthgurram**
## Proposed changes

[//]: # (Renamed appium 'reset' command to 'resetApp' to match with the Appium commands)

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

**Review by hemantjanrao — APPROVED** [review]

**Comment by christian-bromann** [comment]
@srikanthgurram thanks for making a PR. As I mentioned in the issue let's keep the existing command to not break peoples test, we can clean this up in our next major release then,

**Comment by srikanthgurram** [comment]
Thank you @christian-bromann. I will close the PR

**Comment by christian-bromann** [comment]
@srikanthgurram I was not suggesting to discard the PR. I wanted to say to keep the command and add a new command with the same information just calling it `resetApp` this time. Maybe worth also deprecating the current command and adding a deprecation note.

**Comment by srikanthgurram** [comment]
Sorry, I misunderstood. 
Thank you. I will make the changes and re-submit the PR.

**Review by hemantjanrao — APPROVED** [review]

**Comment by christian-bromann** [comment]
@srikanthgurram this looks good, would you mind adding a deprecation notice to the `reset` command?

**Comment by srikanthgurram** [comment]
Hi @christian-bromann, added depreciated notice. Could you please check?

File: packages/wdio-protocols/protocols/appium.json [hunk 1]
```diff
@@ -927,6 +927,23 @@
   "/session/:sessionId/appium/app/reset": {
     "POST": {
       "command": "reset",
+      "description": "@deprecated use 'resetApp' instead.  This is an intern Reset the currently running app for this session.",
```
Comment (christian-bromann) [inline packages/wdio-protocols/protocols/appium.json:930]:
```suggestion
      "description": "<b>Deprecated</b> use 'resetApp' instead. This command will be removed with the upcoming major version release.",
```

**Review by christian-bromann — APPROVED** [review]
👍

**Pull request merged**